package.json

```diff
@@ -56,6 +57,12 @@
                 }
             }
         },
+        "debuggers": [
+            {
+                "type": "lambda-node",
+                "label": "Lambda"
```

Comment:
How does this work with other languages, eg C# and Python. Does the entry grab the label 'Lambda' for its own use or is it OK to add other types with the same label (my guess is that it is ok, just want to be sure).

Reply:
We should have a different label for each language--we don't want to pollute the user's config with configs for environments that they aren't using (it would also result in user confusion when they try a config, but it doesn't work due to an environment mismatch).

So this label should really be `"Lambda (node)"`.

Reply:
✔️